### PR TITLE
Rework flags and bos-loader integration

### PR DESCRIPTION
**Pull request by mpeterdev**
This PR is functionally complete, but awaiting further testing before final cleanup

- moved flag management to bottom panel
- overhauled bos-loader integration to leverage `useWebEngineSandbox` and consolidate our local development flows

in `apps/web/src/pages/[[...root]].tsx`
Screenshot 2024-03-08 at 12 57 29 PM
it would be great if we could combine these into a single hook in the future with the sandbox features as a config option

## UI
- icon in header appears when a bos-loader url is set and shows the URL in a tooltip on hover
- row for flags at top of bottom panel. `showContainerBoundaries` can now be toggled without reload
  - if `showContainerBoundaries` is provided in the URL, that overrides the flag

pr

### CODE REVIEW

**Review by andy-haynes — CHANGES_REQUESTED** [review]
Looks great, nice consolidation!

Scrolling doesn't work for me, either in the code window or within the Components. That's the only blocker I can see.

How much work would it be to convert the bottom panel to tabs? Flags/config could be one, and code could have a dedicated tab. Then I have a clean way to add messages back 😄

> if `showContainerBoundaries` is provided in the URL
I'm not opposed to just killing this URL param...

**Comment by calebjacob** [comment]
Nice, love the new UX! Like Andy said, there are some issues with scrolling that need work.

File: apps/web/src/components/Inspector.module.css [hunk 1]
```diff
@@ -55,17 +56,88 @@
...
+  }
+}
+
+.sourceViewer {
+  display: flex;
+}
+
+.flagSet {
+  display: flex;
+  flex-direction: row;
```
Comment (pavelisnear) [inline apps/web/src/components/Inspector.module.css:87]: Looks like we could remove this line, since this is a default value for the `flex-direction`.

**Comment by mpeterdev** [comment]
> I'm not opposed to just killing this URL param...

it's helpful so that we can automatically enable it when sending developers to the trust demo

**Comment by mpeterdev** [comment]
This is ready to go

Updated UI:
Screenshot 2024-03-18 at 8 10 05 PM
Screenshot 2024-03-18 at 8 10 18 PM
Screenshot 2024-03-18 at 8 10 44 PM
Screenshot 2024-03-18 at 8 10 54 PM

**Review by andy-haynes — APPROVED** [review]
Looks great, nice work!

**Pull request merged**